scripts/unit-test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+source $OKTA_HOME/$REPO/scripts/setup.sh
+
+cd $OKTA_HOME/$REPO
+
+export TEST_SUITE_TYPE="junit"
+export TEST_RESULT_FILE_DIR="${REPO}/src/v3/build2/reports/unit"
+echo $TEST_SUITE_TYPE > $TEST_SUITE_TYPE_FILE
+echo $TEST_RESULT_FILE_DIR > $TEST_RESULT_FILE_DIR_FILE
```

Comment:
Have you all had a chance to test this with a failure? You should see, in the "View Logs" area, an entry under "Infra logs" that's a zip file with the contents of these types of files. I've often seen this misconfigured and, while the test suite fails, the results of these files are not available, so want to make sure this is all working as expected.

Reply:
Ohh good point, I took this config from SIW which does correctly report failures, however, it will be much better to verify, I will break a test to ensure it reports correctly 🙏